roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +54,13 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    @staticmethod
```

Comment:
I would do that too; but pylint really doesn't like it. So it would be:

    def is_active(self):   # pylint: disable=unused-argument,no-self-use
          """Returns true if this check applies to the ansible-playbook run."""
          return True

To me that's more ugly than just letting it be static (which, frankly, it is at the base).

Or if we really wanted to tighten it up:

    is_active = lambda self: True

:-P

That said, I'll change it back if you'd still prefer.